### ISSUE TITLE
NVDA is not announcing the sorting information when focus is on the table headers

### ISSUE THREAD

**Issue by SyedRehmanRazvi** [microsoft/fluentui#13822]
NVDA is not announcing the sorting information when focus is on the column headers.

Repro Steps:
1. Open the https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist
2. Navigate to column headers at the table regions and activate the column headers and verify whether NVDA is announcing the sorting information or not.

Actual Result:
NVDA is not announcing the sorting information on activating the column headers.

Expected Result:
NVDA should announce the sorting information on activating the column headers.

![Sorting info](https://user-images.githubusercontent.com/53210862/85853462-2b8d9100-b7d0-11ea-9419-293cacdc3c80.png)

**Comment by paulgildea** [comment]
@SyedRehmanRazvi which version of Fluent/Fabric are you using in the above screen shot? Thanks!

**Comment by BharathShanker08** [comment]
we are using this URL : https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist to verify
